### ISSUE TITLE
Publish latest version

### ISSUE THREAD

**Issue by mebibou** [weixsong/elasticlunr.js#85]
The latest version available via npm is 0.9.5 but there seems to be a 0.9.6 version which supports `index.search({})`, could it be published?

**Comment by thetutlage** [comment]
Yeah `0.9.6` is some great changes and will be nice to have it on npm. I am currently using the github repo to work with

**Comment by htkoca** [comment]
@weixsong is this project active? I imagine a lot of the issues are related to this. A workaround for the time being is `npm install weixsong/elasticlunr.js#v0.9.6` but that's not ideal.

**Comment by rsj** [comment]
same here - needed for the language support that looks like it was added in `0.9.6`
